consensus/types.go

```diff
@@ -158,7 +158,8 @@ type FastHotStuffEventLoop struct {
 	// Signifies whether or not the event loop has constructed a QC or timeout QC for the current view.
 	// This is an optimization that is useful to prevent the event loop from signaling the server multiple
 	// times for the same view.
```

Comment:
Does this comment apply to both variables? Do we need update it at all?

Reply:
Good catch. I need to update this comment to describe both fields now.